### PR TITLE
admission-control flag replaced by enable/disable-admission-plugins flags

### PR DESCRIPTION
**Pull request by aburdenthehand**
This PR relates to Issue #7235 
Language is moving away from 'admission controllers' to 'admission plugins', but still in transition. 
Order of plugins is not relevant in 1.10
Removed some additional version-specific info in the list plugins (as not relevant for >1.10)

There are two generated files that remain oustanding:
reference/generated/federation-apiserver.md
reference/generated/kube-apiserver.md
As I understand it, these will need to be modified in a separate PR

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7449)

### CODE REVIEW

**Comment by heckj** [comment]
👋 @errordeveloper, @mikedanese, @wlan0, or @davidopp - would love a technical/accuracy review from one of you, if good - please add a `/lgtm` comment to the PR

File: docs/admin/extensible-admission-controllers.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ the following:
 * They need to be compiled into kube-apiserver.
 * They are only configurable when the apiserver starts up.
 
-1.7 introduces two alpha features, *Initializers* and *External Admission
+1.7 introduced two alpha features, *Initializers* and *External Admission
```
Comment (zhangxiaoyu-zidif) [inline docs/admin/extensible-admission-controllers.md]: if here using `-ed`, many other places should be modified too

**Comment by heckj** [comment]
@aburdenthehand this needs a rebase - I looked at this, but wasn't familiar enough to know the correct rebase for the 1.10 content here. Could you rebase please to resolve the merge conflict?

**Comment by aburdenthehand** [comment]
@heckj should be rebased now. Cheers!
(Sorry for the delay - a medical issue unexpectedly kept me offline)

**Pull request merged**